run_client_ssh.ps1

```diff
@@ -0,0 +1,59 @@
+# Connects to the instance and starts OrbitClientGgp. To also deploy OrbitClientGgp,
```

Comment:
I think a license header is missing here.

Reply:
You are right, added!